### DISCUSSION TITLE
Is there a way to use NORMAL MODE in command mode?

### DISCUSSION THREAD

**Discussion by bouillerot** [helix-editor/helix#8456] [Q&A] [answered]
Hello, 
from an old vim-er dinosaur ! :1st_place_medal: 

# I'm trying to use Helix, and I was wondering : 

## “How to add a pattern at the end of each of my text file ?”

Let's say the pattern is to add a comma at the end of each line.

In vim, I would type :+1: 
```vim
:%norm A,
```

After some attempts, I can't find a way to use an equivalent of the command 'norm' as in Vim.

## The questions are : 

- Does that command exists ? 
- How to be in Normal mode, or any other specific mode, via command line ?

**Comment by pascalkuthe** [comment, marked as answer]
helix doesn't allow these kinds of command-based input instead we use multcursor.

You example would be accomplished with `%<a-s>A,` which adds a cursor on each line, and then appends something at the end of line just you would with a single cursor.

**Reply by bouillerot** [reply]
That's crystal clear. I must change my way of thinking a give favor to multicursor mode.
